engine/installation/linux/docker-ee/ubuntu.md

```diff
@@ -42,9 +42,10 @@ To install Docker EE, you need the 64-bit version of one of these Ubuntu version
 - Xenial 16.04 (LTS)
 - Trusty 14.04 (LTS)
 
-Docker EE is supported on `x86_64` (or `amd64`) and `s390x` (IBM Z) architectures.
+Docker EE is supported on `x86_64` (or `amd64`), `s390x` (IBM Z), and `ppc64el`
+(IBM POWER) architectures.
```

Comment:
Discussing with @estesp briefly, and apparently branding switched a few times between `POWER` and `Power`, but current branding for the platform is `Power`; https://www.ibm.com/power